Prepare 3.0.0-rc.5

philfuster opened the pull request:
This PR makes the necessary changes to release a new version `3.0.0-rc.5`. Included are updates to the following:

- `package.json` and `package-lock.json` versions
- Updates to CHANGELOG describing changes in this release
- Updates to documentation versions

shawnmcknight commented on CHANGELOG.md:
The index at the bottom of the changelog needs updating as well.

See https://github.com/STORIS/mvom/pull/755/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR494 for an example.

shawnmcknight commented on CHANGELOG.md:
```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- New `RecordNotFoundError` error was created to be thrown when a record cannot be found in the database. [#740](https://github.com/STORIS/mvom/pull/740)
+- New Increment operation is now supported on numeric fields. The operation will increment the value at the path specified by the amount specified. [#750](https://github.com/STORIS/mvom/pull/750)
+- New internal unibasic subroutine `writeRecord` was created to be shared by `save` and `increment` operations for writes to the database. [#756](https://github.com/STORIS/mvom/pull/756)
+- `increment` operation returns both the original and updated document. [#757](https://github.com/STORIS/mvom/pull/757)
```
757 and 758 can be rolled into 750. They are all part of the same change from the perspective of a reader of this doc.

See how 697, 720, and 732 were grouped together.

shawnmcknight commented on website/docusaurus.config.ts:
```diff
@@ -40,7 +40,7 @@ const config: Config = {
 					versions: {
 						current: {
-							label: '3.0.0-rc.4',
+							label: '3.0.0-rc.5',
```
It doesn't need to precede this PR merging since its a non-breaking change, but the documentation will need to be updated to reflect increment.

shawnmcknight reviewed (changes requested):
Looks good. I had a few small notes.

philfuster commented:
> Looks good. I had a few small notes.

Thank you for the review. I have addressed your feedback in the latest series of commits. I'll have a separate documentation PR done for increment

shawnmcknight reviewed (approved):
Looks good. I will publish a new release now.
